modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -39,15 +39,15 @@ spec:
           pod: {}
         - name: bridge-net <1>
```

Comment:
Are you sure it's correct to give this call-out the same index as the one in spec.template.spec.domain.devices.interface (line 35)? They both must have the same value, but they are 2 different keys in the VM yaml (however, if this is a common practice in documentation - please ignore this comment).